parm/config/gfs/config.ufs

```diff
@@ -171,20 +187,23 @@ case "${fv3_res}" in
           export NEST_DLON=0.125
           export NEST_DLAT=0.125
           export psl_gwd_dx_factor=6.0
-          export WRITE_GROUP_GDAS=2
-          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS=15
-          export WRITE_GROUP_GFS=2
-          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=15
         else
+          if [[ "${RUN}" = *gdas ]]; then
+            export layout_x=4
+            export layout_y=6
+            export nthreads_fv3=1
+            export nthreads_ufs=1
+            export WRITE_GROUP=1
+            export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=10
+          elif [[ "${RUN}" = *gfs ]]; then
+            export layout_x=4
+            export layout_y=6
+            export nthreads_fv3=2
+            export nthreads_ufs=2
+            export WRITE_GROUP=2
+            export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=5
+          fi
```

Comment:
Since the layouts are the same for `gdas` and `gfs` RUNs, they could be pulled out of the if-elif block:

```suggestion
          export layout_x=4
          export layout_y=6
          if [[ "${RUN}" = *gdas ]]; then
            export nthreads_fv3=1
            export nthreads_ufs=1
            export WRITE_GROUP=1
            export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=10
          elif [[ "${RUN}" = *gfs ]]; then
            export nthreads_fv3=2
            export nthreads_ufs=2
            export WRITE_GROUP=2
            export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=5
          fi
```